packages/embark-process-logs-api/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "embark-process-logs-api",
+  "version": "4.1.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Adds various configs to embark's contract.js config file",
```

Comment:
Should this line match the description text in README?

Reply:
My bad. Good catch